spec/Powertrain/Battery.vspec

```diff
@@ -103,7 +103,7 @@
 - Charging.ChargePortFlap:
   datatype: string
   type: actuator
-  value: closed
+  default: closed
```

Comment:
So "default" can be used for all types (actuators7sensors/attributes)? Does the documentation need to be updated, it seems to mention default only for attributes:
https://genivi.github.io/vehicle_signal_specification/rule_set/data_entry/sensor_actuator/ 
https://genivi.github.io/vehicle_signal_specification/rule_set/data_entry/attributes/